the-experts-BE/app.js

```diff
@@ -3,7 +3,7 @@ const path = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
 const getContractor = require("./routes/contractor-results");
-const addContractor = require("./routes/contractor-post");
+const queries = require("./queries/index");
```

Comment:
You can just say `require("./queries");` - If you have a file called `index.js` inside a folder, node will look for that automatically so you just need to say the folder name.